build.gradle

```
@@ -96,6 +86,11 @@ dependencies {
     // expiring map
     implementation 'net.jodah:expiringmap:0.5.10'
 
+    // tests
+    testImplementation "org.springframework.boot:spring-boot-starter-test"
+    testRuntimeOnly("org.junit.platform:junit-platform-launcher")
```

Should we stay consistent with the other lines?
```suggestion
    testRuntimeOnly 'org.junit.platform:junit-platform-launcher'
```

---

We follow gradle upgrade guide here
https://docs.gradle.org/current/userguide/upgrading_version_8.html#manually_declaring_dependencies